### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Pirakas Anthon, Aurélien Richard, Romain Humair and Lucas Lattion

### DISCUSSION THREAD

**Discussion by lucaslattion** [heig-vd-dai-course/heig-vd-dai-course#190] [Show and tell]
# DAI 2023-2024 - Practical work 4 - Pirakas Anthon, Aurélien Richard, Romain Humair and Lucas Lattion

**Comment by apirakas** [comment]
Repo : https://github.com/Lyrafll/DAI-Practical-Work-4
Docker file: https://github.com/users/apirakas/packages/container/package/dai-pw4
Webiste: https://dai-pw4.ddns.net/

**Reply by HumairRomain** [reply]
Last Hash is 382adac39c74389204b8881c42e5dba03b42aee1
[https://github.com/Lyrafll/DAI-Practical-Work-4/tree/382adac39c74389204b8881c42e5dba03b42aee1](url)
